src/components/PageHeader/_page-header.scss

```diff
@@ -62,6 +62,7 @@
 .cc-page-header__standfirst {
   font-size: var(--heading-md);
   margin-bottom: var(--space-xl);
+  text-align: center;
```

Comment:
We seem to be flip-flopping on the alignment of this. It was previously centre aligned but then a design decision was made to left align it. Please can you double check this?

Reply:
from Laura: `There has been misunderstandings on alignment previously, everything should still be center aligned for the foreseeable. `